flow/src/main/resources/schema/flow/20220713094831_CB-17459_add_jackson_serialized_target_columns.sql

```diff
@@ -0,0 +1,17 @@
+-- // CB-17459 add columns to store Jackson-serialized objects too
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE IF EXISTS flowlog ADD COLUMN IF NOT EXISTS payloadjackson text;
+ALTER TABLE IF EXISTS flowlog ADD COLUMN IF NOT EXISTS variablesjackson text;
+
+ALTER TABLE IF EXISTS flowchainlog ADD COLUMN IF NOT EXISTS triggereventjackson text;
+ALTER TABLE IF EXISTS flowchainlog ADD COLUMN IF NOT EXISTS chainjackson text;
```

Comment:
* The entity field `FlowChainLog.chainJackson` is marked as `nullable = false`, but there is no corresponding `NOT NULL` modifier here.
* If the `nullable = false` restriction is indeed valid, I wonder how this column will be initialized for existing records. Will it be set to the empty string?

Reply:
I guess I cannot create a NOT NULL column on an existing non-empty table as postgres will respond with `SQL Error [23502]: ERROR: column "triggereventjackson" of relation "flowchainlog" contains null values.`
The nullable restriction is valid from backend point of view i.e. I assume Hibernate won't allow null values into this column.

Reply:
Althouht I need to change the `nullable = false` to `true` because the table will indeed contain null valued fields (all the previous flowhainlogs before change).